types/hump/camera.d.tl

```diff
@@ -0,0 +1,39 @@
+local record camera
+
+    type DrawFunction = function()
+    type SmoothingFunction = function(x: number, y: number, s: number): number, number
+
+    new: function(x: number, y: number, zoom: number, rot: number): camera
```

Comment:
Apparently, it is also possible to create a new camera by calling the module as a function:

```lua
    metamethod __call: function(self: camera, x: number, y: number, zoom: number, rot: number): camera
```

Reply:
ah right i missed that part for the camera, reckon it worked the same as the vectors

Reply:
added